modules/tinymce/src/themes/silver/main/ts/ui/menus/contextmenu/Settings.ts

```diff
@@ -29,7 +29,7 @@ const getMenuItems = (editor: Editor, name: string, defaultItems: string): strin
 const isContextMenuDisabled = (editor: Editor): boolean => editor.getParam('contextmenu') === false;
 
 const getContextMenu = (editor: Editor): string[] => {
-  return getMenuItems(editor, 'contextmenu', 'link linkchecker image imagetools table spellchecker configurepermanentpen');
+  return getMenuItems(editor, 'contextmenu', 'link linkchecker image table spellchecker configurepermanentpen');
```

Comment:
Just wanted to check if this should have been removed? Could be misunderstanding the code, but we still want imagetools to show in the default context menu even when it becomes a premium plugin?

Reply:
correct

- [ ] revert change